api/ruby/ghe-org-permissions-report.rb

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env ruby
+# Generates a CSV report listing all organizations, their repositories,
+# collaborators, effective permissions, teams, and team permissions
+#
+# Set OCTOKIT_ACCESS_TOKEN to a token with read:org scope owned by a site admin
+# and OCTOKIT_API_ENDPOINT to http(s)://[your-hostname]/api/v3/
```

Comment:
We should probably provide [the `Octokit.configure`](https://github.com/octokit/octokit.rb#interacting-with-the-githubcom-apis-in-github-enterprise) setup here too with this env var.